app/models/authenticator/httpd_oauth2.rb

```diff
@@ -0,0 +1,162 @@
+module HttpdOauth2
```

Comment:
This should under the Authenticator namespace.

Reply:
Actually, since this is really a submodule just for Authenticator::Httpd, I wonder if this should be Authenticator::Httpd::Oauth2 (and correspondingly live in app/models/authenticator/httpd/oauth2.rb)